ui/src/app/views/app/component.ts

```diff
@@ -102,4 +105,30 @@ export class AppComponent implements OnInit {
     }
   }
 
+  /**
+   * Setting the Google Analytics ID here.
+   * This first injects Google's gtag script via iife, then secondarily defines
```

Comment:
What is iife?

Reply:
Immediately Invoked Function Expression
http://benalman.com/news/2010/11/immediately-invoked-function-expression/

Reply:
And for those of you around at the time of that article, I was at the jQuery conference (that year) where Ben had a session on this subject. Great speaker.